### PR TITLE
Databricks: Test code & Train with created dataset

### PR DESCRIPTION
**Pull request by markus-hinsche**
**User stories:**
- https://dev.azure.com/cgmorg/ChildGrowthMonitor/_workitems/edit/1306
- https://dev.azure.com/cgmorg/ChildGrowthMonitor/_workitems/edit/1459

**Changes:** 
- Add unit tests for `mlpipeline_utils.py`
- Fix databricks dataset creation

**Result**
- I created a dataset with 100 / 5000 artifacts
- I can use this dataset created by Databricks to train a plaincnn model for a few epochs (locally and on cluster, see [Run](https://ml.azure.com/runs/training-junkyard_1623873713_c267dc8b?wsid=/subscriptions/9b5bbfae-d5d1-4aae-a2ca-75159c0c887d/resourcegroups/cgm-ml-prod-we-rg/workspaces/cgm-ml-prod-we-azml&tid=3a27c573-ec1a-4734-9cd3-3208af51794b))

### CODE REVIEW

File: src/common/data_utilities/tests/test_mlpipeline_utils.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+
+def test_get_depthmaps():
+    zip_input_full_path = DATA_UTILITIES_DIR / 'tests' / 'zip_files' / ARTIFACT_ZIP_PATH
+    preprocessed_depthmaps = get_depthmaps([zip_input_full_path])
+    assert preprocessed_depthmaps[0].shape == (180, 240, 1), preprocessed_depthmaps[0].shape
+
+
+if __name__ == '__main__':
+    test_load_depth_hugh()
```
Comment (gangooteli) [inline src/common/data_utilities/tests/test_mlpipeline_utils.py:64]: Can you include below test cases also:
test_artifact_processor()
test_get_depthmaps()

**Review by gangooteli — COMMENTED** [review]
Can you fill the PR template which we use in cgm-ml?

File: src/common/data_utilities/tests/test_mlpipeline_utils.py (hunk 1 above)
Comment (markus-hinsche) [inline src/common/data_utilities/tests/test_mlpipeline_utils.py:64]:
```suggestion


if __name__ == '__main__':
    test_load_depth_hugh()
```

Comment (markus-hinsche) [inline src/common/data_utilities/tests/test_mlpipeline_utils.py:64]:
```suggestion
```

Comment (markus-hinsche) [reply src/common/data_utilities/tests/test_mlpipeline_utils.py]: Ohh nice catch, this code was just for debugging purposes, removed it. 
These tests will be found by pytest without the main block

**Review by gangooteli — APPROVED** [review]

**Pull request merged**